src/raw_node.rs

```diff
@@ -209,7 +224,8 @@ struct ReadyRecord {
 pub struct LightReady {
     commit_index: Option<u64>,
     committed_entries: Vec<Entry>,
-    messages: Vec<Vec<Message>>,
+    messages: Vec<Message>,
+    is_persisted_msg: bool,
```

Comment:
Probably should be stored and checked in `Ready`. `take_persisted_messages` is meaningless for `LightReady`.

Reply:
Done